.codecov.yml

```diff
@@ -0,0 +1,22 @@
+ignore:
+#  - "path/to/folder"  # ignore folders and all its contents
+#  - "test_*.rb"       # wildcards accepted
+#  - "**/*.py"         # glob accepted
+#  - "[a-z]+/test_.*"  # regexp accepted
+
+  # Setup configuration
+  - '**/psic/setup.py'
+
+  # Test files
+  - '**/psic/tests'
+
+  # Command-line runners
+  - '**/psic/assigner/assign.py'
+  - '**/psic/cataloging/catalog.py'
+  - '**/psic/collector/collect.py'
+  - '**/psic/resizer/resize.py'
```

Comment:
This is a test review comment :)

As a note, I chose to ignore these files in coverage because they are essentially command-line parsers that run the actual logic that is found in other files, so it's not practical to try and unit test them.

Reply:
test comment wowe